### DISCUSSION TITLE
schematics or wiring diagrams

### DISCUSSION THREAD

**Discussion by Baerenwolf** [mrbeam/MrBeamPlugin#1843] [General]
Sorry to bother you in tough times, i don´t even know if this is still active.

I came in contact with Mr.Beam in a Makerspace north of Munich. Now we wonder how we will get spare parts in the future.
Is it possible to get some Schematics or wiring of the machine.
I would love to keep the machine running as long as possible, its a wonderful kit to get some People started with Crafting.

I really like your idea with Mr.Beam, i considered retrofitting another laser, but i came across a post were you said that usage with another laser was not planned. 
I would love to make it somewhat possible, but i don´t have the time or the skills to adapt everything, maybe some Documentation would save me a really substantial amount of time. 
Thank you for your work! Thats some nice to use machine. 
Maybe i can dissassemble it sometime soon and learn from that, but as long as it works i dont really want to do that. 

Thank you in advance for your answer.
Greetings
Bärenwolf

**Comment by hungerpirat** [comment]
You can get spare parts by writing to ***@***.***
